resources/services/cloudformation/stacks.go

```diff
@@ -0,0 +1,335 @@
+package cloudformation
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/cloudformation"
+	"github.com/aws/aws-sdk-go-v2/service/cloudformation/types"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+//go:generate cq-gen --resource stacks --config gen.hcl --output .
+func Stacks() *schema.Table {
+	return &schema.Table{
+		Name:         "aws_cloudformation_stacks",
+		Description:  "The Stack data type.",
+		Resolver:     fetchCloudformationStacks,
+		Multiplex:    client.ServiceAccountRegionMultiplexer("cloudformation"),
+		IgnoreError:  client.IgnoreAccessDeniedServiceDisabled,
+		DeleteFilter: client.DeleteAccountRegionFilter,
+		Options:      schema.TableCreationOptions{PrimaryKeys: []string{"id"}},
+		Columns: []schema.Column{
+			{
+				Name:        "account_id",
+				Description: "The AWS Account ID of the resource.",
+				Type:        schema.TypeString,
+				Resolver:    client.ResolveAWSAccount,
+			},
+			{
+				Name:        "region",
+				Description: "The AWS Region of the resource.",
+				Type:        schema.TypeString,
+				Resolver:    client.ResolveAWSRegion,
+			},
+			{
+				Name:     "arn",
+				Type:     schema.TypeString,
+				Resolver: schema.PathResolver("StackId"),
+			},
```

Comment:
This field seems to be duplicated here and in `id`

Reply:
I added ARN for the sake of having an ARN column, even though the StackId is actually the ARN. maybe in a future version I can remove from stakcId all the arn part and just keep the UUID.